File: Python_Assignment/Python_Programming_Assignment_2/Assignment2_1.py
# 1. Create on module named as Arithmetic which contains 4 functions
#    as Add() for addition, Sub() for substraction, Mult() for multiplication
#    and Div() for division. All functions accepts two parameters as number
#    and perform the operation. Write on python program which call all the
#    functions from Arithmetic module by accepting the parameters from user.

# Importing module Arithmetic
import Arithmetic

def main():
    No1 = int(input("Enter first number :"))
    No2 = int(input("Enter second number :"))

    print("Addition is : ", Arithmetic.Add(No1, No2))
    print("Substraction is : ", Arithmetic.Sub(No1, No2))
    print("Multiplicaton is : ", Arithmetic.Mult(No1, No2))
    # to avoid divide-by-zero error
    if No2 == 0:
        print("Divisin of zero not allowed")
    else:
        print("Division is : ", Arithmetic.Div(No1, No2))


if __name__ == "__main__":
    main()

File: Typescript_Assignment/Typescript_Assignment2/Assignment2_1.js
// 2.1 Write a typescript program which contains one funciotn named as Maximum.
//     That function accepts array of numbers andd returns the largest number from array
// Input : 23 89 6 29 56 45 77 32
// Output : Maximum number is 89
function Maximum(list) {
    var temp = list[0];
    for (var i = 0; i < list.length; i++) {
        if (temp < list[i]) {
            temp = list[i];
        }
    }
    return temp;
}
var Data = new Array();
Data = [23, 89, 6, 29, 56, 45, 77, 32];
console.log("Maximum number is " + Maximum(Data));
